lib/typed_ecto_schema/ecto_type_mapper.ex

```diff
@@ -192,7 +194,8 @@ defmodule TypedEctoSchema.EctoTypeMapper do
        do: false
 
   defp field_is_nullable?(_default, function_name, _args)
-       when function_name in @schema_assoc_function_name,
+       when function_name in @schema_assoc_function_name and
+              function_name not in @schema_belongs_to_assocs,
```

Comment:
Wouldn't it is easier to do something like `@schema_many_assoc_function_name` and then this could be just `when function name in @schema_many_assoc_function_name` or something like that?

Reply:
Makes sense, updated it!